src/modules/esl-utils/dom/events/target.ts

```diff
@@ -9,12 +9,23 @@ export class SyntheticEventTarget implements EventTarget {
 
   private readonly _listeners: Record<string, EventListenerOrEventListenerObject[]> = {};
 
+  protected getEventListeners(): EventListenerOrEventListenerObject[];
+  protected getEventListeners(type: string): EventListenerOrEventListenerObject[];
+  protected getEventListeners(type?: string): EventListenerOrEventListenerObject[] {
+    if (!type) return Object.values(this._listeners).flat(1);
```

Comment:
1) the collection that we return should be unique
2) as this is not beta and the current target is 4, we can not overuse ES6 built-in methods as officially we still have ie11 until 5th version is released (We can assume Object.values will be covered by polyfill, but as for now flat 
 is better to be consumed from esl-utils/misc/array)